### ISSUE TITLE
In Browse mode, NVDA announce irrelevant information 'Clickable' repeatedly

### ISSUE THREAD

**Issue by shivasai09** [nvaccess/nvda#12090]
### Steps to reproduce:
open new edge chromium browser
open link https://codepen.io/ssarapu/pen/GRNMmry
open NVDA
focus on hello world button
shift to browse mode by clicking `insert + spacebar`
then press down arrow until focus comes to line chart

### Actual behavior:
NVDA announce irrelevant information 'Clickable' repeatedly on graph below 'Behavior impact on compromise rate' user navigate in browse mode

### Expected behavior:
NVDA Should not announce irrelevant information 'Clickable' repeatedly on graph below

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

![image](https://user-images.githubusercontent.com/33802398/108673112-1efd3c00-7509-11eb-8575-d6ed17c37734.png)

**Comment by shivasai09** [comment]
ping

**Comment by ehollig** [comment]
> NVDA will consider reporting clickable if the node has click, mouseDown or mouseUp event
See https://github.com/nvaccess/nvda/issues/5830#issuecomment-199778283
In this case, onclick events are present, causing NVDA to announce clickable.
